src/VisualStudio.props

```diff
@@ -74,6 +81,19 @@
            or '%(FileName)' == 'VSLangProj110'">
         <EmbedInteropTypes>true</EmbedInteropTypes>
       </ReferencePath>
+      <ReferencePath Condition="
```

Comment:
Add comment explaining why this is necessary

Reply:
Fixed.

Reply:
Comparing binaries before and after confirms that this doesn't change the set of things that are embedded?